### PR TITLE
Lesson 1 1

### PR DESCRIPTION
**Pull request by annaUniversal**

### CODE REVIEW

**Comment by JoHeyJo** [comment]
@annaUniversal Thanks for submitting a PR. This way I can clearly annotate the code we're reviewing. Please, take a look at "Files changed". I'll elaborate on your assignment.

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,28 @@
 import React from 'react';
```
Comment (JoHeyJo) [inline src/App.js:2]: `var` generally doesn't get used unless anymore unless for very specific reasons. There are two other declarations that are more commonly used. Do a little research and see which one better fits your use case.

File: src/App.js [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+  return(
```
Comment (JoHeyJo) [inline src/App.js:19]: There is some data missing. What kind of list is this?

File: src/App.js [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
+  <ul>
+    {todoList.map( (item) =>
+       <li key={item.id}>{item.title}</li>
+    )}
```
Comment (JoHeyJo) [inline src/App.js]: Great variable naming! You always want your variables to be as descriptive and specific as possible.

File: src/App.js [hunk 4]
```diff
@@ -1,18 +1,28 @@
...
+    id: 2,
+    title: "Take a shower"
+  },
+  {
+    id: 3,
+    title: "Walk the dog"
+  }
+];
+
 function App() {
```
Comment (JoHeyJo) [inline src/App.js:18]: Don't forget to format your code. It's going to make reading code a lot easier especially as it grows. Your IDE should have a friendly 'hotkey' or in settings a command. It's possible you might need to download an extension to provide formatting if it doesn't come out of the box.

File: src/App.js (hunk 1 above)
Comment (annaUniversal) [reply src/App.js:2]: changed to let

File: src/App.js (hunk 2 above)
Comment (annaUniversal) [reply src/App.js:19]: Not sure what you are talking about. But I added the header.

Comment (JoHeyJo) [reply src/App.js:19]: That's exactly what I meant. Thank you!

**Comment by annaUniversal** [comment]
Initiated ToDo list

**Pull request merged**